From 5ce7c55c5050405044b2a2f1359ffdc5be832a99 Mon Sep 17 00:00:00 2001
From: "Hebbar, Gururaja" <gururaja.hebbar@ti.com>
Date: Tue, 28 Feb 2012 14:12:56 +0530
Subject: [PATCH 0871/1092] ARM: OMAP2+: am33xx: update mcasp platform
 registration

McASP device registration now uses omap_device api. This change needs a
new argument (device id). This patch changes the call to involve device
id along with platform data.

Signed-off-by: Hebbar, Gururaja <gururaja.hebbar@ti.com>
---
 arch/arm/mach-omap2/board-am335xevm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-omap2/board-am335xevm.c b/arch/arm/mach-omap2/board-am335xevm.c
index e18ef06..3919dab 100644
--- a/arch/arm/mach-omap2/board-am335xevm.c
+++ b/arch/arm/mach-omap2/board-am335xevm.c
@@ -1434,7 +1434,7 @@ static void mcasp1_init(int evm_id, int profile)
 {
 	/* Configure McASP */
 	setup_pin_mux(mcasp1_pin_mux);
-	am335x_register_mcasp1(&am335x_evm_snd_data1);
+	am335x_register_mcasp(&am335x_evm_snd_data1, 1);
 	return;
 }
 
-- 
1.7.11.2

